client/src/reducers/genesets.ts

```diff
@@ -24,9 +24,28 @@
  */
 import { diffexpPopNamePrefix1, diffexpPopNamePrefix2 } from "../globals";
 
+interface Gene {
+  geneSymbol: string;
+  geneDescription: string;
+}
+
+interface Geneset {
+  genesetName: string;
+  genesetDescription: string;
+  genes: Map<Gene["geneSymbol"], Gene>;
+}
+
+export type Genesets = Map<Geneset["genesetName"], Geneset>;
+
+interface State {
```

Comment:
export as less generic `GenesetsState`

Reply:
Also type action

Reply:
Ah I didn't export State because no one is importing it at the moment, maybe we can keep it not exported until a callsite actually needs it?

Just added `AnyAction` for `action` 🙆‍♂️